### PR TITLE
fix: wadouri metadata was not using scaling parameters properly

### PR DESCRIPTION
**Pull request by sedghi**
fixes https://github.com/cornerstonejs/cornerstone3D-beta/issues/153 and https://github.com/cornerstonejs/cornerstone3D-beta/issues/107

https://github.com/cornerstonejs/cornerstoneWADOImageLoader/pull/464 should be merged first

### CODE REVIEW

**Comment by sedghi** [comment]
https://github.com/cornerstonejs/cornerstone3D-beta/issues/153

File: packages/core/src/RenderingEngine/StackViewport.ts [hunk 1]
```diff
@@ -314,6 +315,7 @@ class StackViewport extends Viewport implements IStackViewport {
       },
       scalarData: this.cpuImagePixelData,
       hasPixelSpacing: this.hasPixelSpacing,
+      image: image,
```
Comment (sedghi) [inline packages/core/src/RenderingEngine/StackViewport.ts:318]: remove this

File: packages/core/src/types/IImageData.ts [hunk 2]
```diff
@@ -24,6 +24,23 @@ interface IImageData {
   scaling?: Scaling;
   /** whether the image has pixel spacing and it is not undefined */
   hasPixelSpacing?: boolean;
+  /** preScale object */
+  preScale?: {
+    enabled?: boolean;
```
Comment (sedghi) [inline packages/core/src/types/IImageData.ts:29]: no need for enabled here

**Review by swederik — APPROVED** [review]
LGTM when the latest WADO Image Loader is added

**Pull request merged**